src/main/scala/io.circe.config/package.scala

```diff
@@ -54,6 +59,11 @@ import scala.collection.JavaConverters._
  * }}}
  */
 package object config {
+  def loadConfigF[F[_], C : Decoder](implicit ev : ApplicativeError[F, Throwable]) : F[C] =
+    ConfigFactory.load().as[C].leftWiden[Throwable].raiseOrPure[F]
+
+  def loadConfigF[F[_], C : Decoder](path : String)(implicit ev : ApplicativeError[F, Throwable]) : F[C] =
+    ConfigFactory.load().as[C](path).leftWiden[Throwable].raiseOrPure[F]
```

Comment:
The way `ConfigFactory.load()` is used here means that load error won't be caught. I suggest moving these methods to the `syntax` module either named `asF` or `loadF`. That way they can piggy-bag on the existing config parsers and simply focus on conversions for `ApplicativeError`.

Reply:
To clarify, my suggest is to add the following code to `io.circe.config.syntax.CirceConfigOps`:
```scala
    import cats.ApplicativeError
    import cats.implicits._

    /**
     * Read config settings into the specified container type.
     */
    def asF[F[_], C: Decoder](implicit ev : ApplicativeError[F, Throwable]) : F[C] =
      as[C].leftWiden[Throwable].raiseOrPure[F]

    /**
     * Read config settings at given path into the specified container type.
     */
    def asF[F[_], C: Decoder](path : String)(implicit ev : ApplicativeError[F, Throwable]) : F[C] =
      as[C](path).leftWiden[Throwable].raiseOrPure[F]
```

Reply:
Sounds good Jonas. Asking mainly because I'm using this line of code in a bunch of projects: what load error do you mean? You mean if `ConfigFactory.load()` throws some exception? I think that method always returns a `Config` so any errors will happen when parsing it with `as`.

Reply:
Ah you're totally right, I get it now. So if there's a syntax error in `application.conf` or something, there will be a parse exception that won't get caught by `.as`. 

Ok, I'll think about it a bit and add your changes. Thanks!

Reply:
Added this in 6d76b92

Reply:
So I took one more stab at single function usage in 8dd4315. I do like the idea of the user not having to import `ConfigFactory`.  Let me know what you think. I can wrap `parseResources` too if that helps, or i can just revert 8dd4315